lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -637,7 +637,7 @@ def _section_send_email(course, access):
     course_modes = []
     from verified_track_content.models import VerifiedTrackCohortedCourse
     if not VerifiedTrackCohortedCourse.is_verified_track_cohort_enabled(course_key):
-        course_modes = CourseMode.modes_for_course(course_key)
+        course_modes = CourseMode.modes_for_course(course_key, include_expired=True, only_selectable=False)
```

Comment:
Expired means the upgrade deadline has passed, and users can no longer enroll in the given course mode. This should (although, I make no promises for older data) only apply to the verified and professional modes. I am uncertain about `only_selectable`.